### PR TITLE
[Zeppelin-1022] Apply new mechanism to LivyInterpreter

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
Apply new mechanism to LivyInterpreter

### What type of PR is it?
[Improvement]

### Todos
* [x] - Apply new mechanism to LivyInterpreter
* [x] - rename zeppelin.livy.url to livy.host.url to make all params look livy.*
* [x] - surround interpreterContext.getAuthenticationInfo().getUser() with ""

### What is the Jira issue?
* [Zeppelin-1022](https://issues.apache.org/jira/browse/ZEPPELIN-1022)

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
CI fails for unrelated changes

**Comment by bzz** [comment]
@prabhjyotsingh if we could point what is the reason of failure and link to either PR or existing JIRA issue that handles it - that could help improve our infrastructure a lot

Looks good to me
\cc @jongyoul as an author of new interpreter registration system for a reivew

File: livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java [hunk 1]
```diff
@@ -75,13 +75,13 @@ public Integer createSession(InterpreterContext context, String kind) throws Exc
 
       String confData = gson.toJson(conf);
 
-      String json = executeHTTP(property.getProperty("zeppelin.livy.url") + "/sessions",
+      String json = executeHTTP(property.getProperty("livy.host.url") + "/sessions",
```
Comment (felixcheung) [inline livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java]: my thought would be this:
- any property needed by Zeppelin (such as to communicate with another server), it should start with "zeppelin.*"
- any property not used by Zeppelin but by a separate server or framework (eg. spark), it should start with "spark.*"
- the reason is the interpreter can then pass-through properties as-is, for example, spark interpreter can pass all "spark.*" property to SparkConf (and skip all the zeppelin.* ones!)

I think we try to roughly follow that pattern (at least as much as I have seen)

**Comment by jongyoul** [comment]
I've tested it. LGTM

File: livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java (hunk 1 above)
Comment (prabhjyotsingh) [inline livy/src/main/java/org/apache/zeppelin/livy/LivyHelper.java]: Fair point, I'll revert this change.

**Comment by prabhjyotsingh** [comment]
@bzz I was referring to https://github.com/apache/zeppelin/pull/1034, ZEPPELIN-1009. 

```
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 104.524 sec - in org.apache.zeppelin.integration.ZeppelinIT

Results :

Tests in error: 
  ParagraphActionsIT.testRemoveButton:153 Â» NoSuchElement Unable to locate eleme...
  ParagraphActionsIT.testCreateNewButton:86 Â» NoSuchElement Unable to locate ele...
```


@jongyoul  Thank you for testing it out.

**Comment by r-kamath** [comment]
@prabhjyotsingh LGTM
>  surround interpreterContext.getAuthenticationInfo().getUser() with ""

Thanks

**Comment by prabhjyotsingh** [comment]
Will merge this if no more discussion.

**Pull request closed without merging**